UnitTests/Views/ViewTests.cs

```diff
@@ -4579,7 +4579,7 @@ public void View_Instance_Use_Attribute_Normal_On_Draw ()
 			TestHelpers.AssertDriverColorsAre (@"
 000000
 011110
-000000", [Colors.TopLevel.Normal, Colors.TopLevel.Focus]);
+000000", new Attribute [] { Colors.TopLevel.Normal, Colors.TopLevel.Focus });
```

Comment:
Maybe there is some misconfiguration on the project or on the server. It seems a problem with csharp language version. Change from collection expression to array collection stopped failing.